test/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  db:
+    build:
+      context: ./postgis_docker
+      dockerfile: ./Dockerfile
+    volumes:
+      - ./postgis_docker/setupdata.sql:/docker-entrypoint-initdb.d/setup-db.sql
+    environment:
+      - POSTGRES_DB=gis
+      - POSTGRES_USER=docker
+      - POSTGRES_PASS=docker
+      - ALLOW_IP_RANGE=0.0.0.0/0
+      - POSTGRES_MULTIPLE_EXTENSIONS=postgis,hstore,postgis_topology,postgis_raster,pgrouting
+      - SSL_CA_FILE=/var/lib/postgresql/root.crt
+      - SSL_KEY_FILE=/var/lib/postgresql/privkey.pem
+      - SSL_CERT_FILE=/var/lib/postgresql/fullchain.pem
+      - IGNORE_INIT_HOOK_LOCKFILE=true
+      - RUN_AS_ROOT=true
+    ports:
+      - "5440:5432"
+    restart: on-failure
+    healthcheck:
+      test: "PGPASSWORD=docker pg_isready -h 127.0.0.1 -U docker -d gis"
+
+  qgis-server:
+    image: qgis/qgis-server:ltr
```

Comment:
Can we use a fixed version here?
I'd rather check for regressions than compat with or issues of latest.